build.gradle

```diff
@@ -12,24 +12,28 @@ buildscript {
     ext.gradleVersion = '3.3.1'
     ext.junitVersion = "4.12"
     ext.koinVersion = "1.0.2"
-    ext.kotlinVersion = "1.3.21"
-    ext.lifecycleVersion = "2.0.0"
+    ext.kotlinVersion = "1.3.41"
+    ext.lifecycleVersion = "2.1.0-rc01"
```

Comment:
Do you really need the `-rc` version?

Reply:
It was one of the @ashdavies suggestions